app/views/shared/_error_messages.html.erb

```diff
@@ -1,4 +1,4 @@
-<% if @object&.errors.any? %>
+<% if  @object && @object.errors.any? %>
```

Comment:
Sao phải sửa code cũ vậy em

Reply:
vì thằng @object nó chưa được load mà lại gắn vào any? thì khi object nó nil thì toàn bộ hàm  @object&.errors.any? cũng sẽ nil luôn ạ ! còn khi dùng như cũ nó sẽ load @object nếu nó nil nó sẽ thực hiện toán tử && ở đây thì nó sẽ trả về false !!

Reply:
@object&.errors&.any?
Thử như thế này thử em nhé